spec.bs

```diff
@@ -1293,7 +1517,12 @@ To <dfn>report result</dfn> given a [=leading bid info=] |leadingBidInfo|:
   1. Let « |sellerSignals|, |reportUrl|, |reportingBeaconMap| » be the result of [=evaluating a
      reporting script=] with |sellerReportingScript|, "`reportResult`", and « |config|,
      |browserSignals|».
-  1. TODO: Store |reportUrl| and |reportingBeaconMap| in the {{FencedFrameConfig}} as appropriate.
+  1. Let |reportingResult| be a [=reporting result=] where [=reporting result/report url=] is
+     |reportUrl| and [=reporting result/reporting beacon map=] is |reportingBeaconMap|.

```

Comment:
Can you fill this struct out with the normal struct construction syntax (`: foo` and `:: bar`). And below too.